exchanges/binanceus/binanceus_types.go

```diff
@@ -0,0 +1,1179 @@
+package binanceus
+
+import (
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/shopspring/decimal"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/asset"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/trade"
+)
+
+var (
+	// BinanceRequestParamsOrderLimit Limit order
+	BinanceRequestParamsOrderLimit = RequestParamsOrderType("LIMIT")
+	// BinanceRequestParamsOrderMarket Market order
+	BinanceRequestParamsOrderMarket = RequestParamsOrderType("MARKET")
+	// BinanceRequestParamsOrderStopLoss STOP_LOSS
+	BinanceRequestParamsOrderStopLoss = RequestParamsOrderType("STOP_LOSS")
+	// BinanceRequestParamsOrderStopLossLimit STOP_LOSS_LIMIT
+	BinanceRequestParamsOrderStopLossLimit = RequestParamsOrderType("STOP_LOSS_LIMIT")
+	// BinanceRequestParamsOrderTakeProfit TAKE_PROFIT
+	BinanceRequestParamsOrderTakeProfit = RequestParamsOrderType("TAKE_PROFIT")
+	// BinanceRequestParamsOrderTakeProfitLimit TAKE_PROFIT_LIMIT
+	BinanceRequestParamsOrderTakeProfitLimit = RequestParamsOrderType("TAKE_PROFIT_LIMIT")
+	// BinanceRequestParamsOrderLimitMarker LIMIT_MAKER
+	BinanceRequestParamsOrderLimitMarker = RequestParamsOrderType("LIMIT_MAKER")
+)
+
+const wsRateLimitMilliseconds = 250
+
+// crypto withdrawals status codes description
+const (
+	EmailSent = iota
+	Cancelled
+	AwaitingApproval
+	Rejected
+	Processing
+	Failure
+	Completed
+)
+
+// ExchangeInfo holds the full exchange information type
+type ExchangeInfo struct {
+	Code       int       `json:"code"`
+	Msg        string    `json:"msg"`
+	Timezone   string    `json:"timezone"`
+	Servertime time.Time `json:"serverTime"`
+	RateLimits []struct {
+		RateLimitType string `json:"rateLimitType"`
+		Interval      string `json:"interval"`
+		Limit         int    `json:"limit"`
+	} `json:"rateLimits"`
+	ExchangeFilters interface{} `json:"exchangeFilters"`
+	Symbols         []struct {
+		Symbol                     string   `json:"symbol"`
+		Status                     string   `json:"status"`
+		BaseAsset                  string   `json:"baseAsset"`
+		BaseAssetPrecision         int      `json:"baseAssetPrecision"`
+		QuoteAsset                 string   `json:"quoteAsset"`
+		QuotePrecision             int      `json:"quotePrecision"`
```

Comment:
```suggestion
		Limit         int64    `json:"limit"`
	} `json:"rateLimits"`
	ExchangeFilters interface{} `json:"exchangeFilters"`
	Symbols         []struct {
		Symbol                     string   `json:"symbol"`
		Status                     string   `json:"status"`
		BaseAsset                  string   `json:"baseAsset"`
		BaseAssetPrecision         int64      `json:"baseAssetPrecision"`
		QuoteAsset                 string   `json:"quoteAsset"`
		QuotePrecision             int64      `json:"quotePrecision"`
```

Reply:
Please update all `int` to `int64`

Reply:
There are still more `int` around. Please update all of the `int` usage in `binanceus_types` to use `int64`. This helps with parsing data for 32 bit applications